### ISSUE TITLE
"connection save" in interactive mode requires current connection

### ISSUE THREAD

**Issue by andy-maier** [pywbem/pywbemtools#395]
Example:
```
$ pywbemcli
Enter 'help' for help, <CTRL-D> or ':q' to exit pywbemcli.
pywbemcli> --server http://localhost --user usr1 --password blah connection save testconn3
Error: No current connection connection. Use "connection list" to see more connection information
pywbemcli> connection list
WBEM server connections: (#: default, *: current)
+-----------+------------------+-------------+--------+-----------+----------+----------------------------------------+
| name      | server           | namespace   | user   |   timeout | verify   | mock-server                            |
|-----------+------------------+-------------+--------+-----------+----------+----------------------------------------|
| bla       | http://blahblah  | root/cimv2  |        |        30 | True     |                                        |
| local     | http://localhost | root/cimv2  | me     |        30 | True     |                                        |
| mockfuzzy |                  | root/cimv2  |        |        30 | True     | tests/unit/simple_mock_model.mof       |
|           |                  |             |        |           |          | tests/unit/simple_mock_invokemethod.py |
| mymock    |                  | root/cimv2  |        |        30 | True     | tests/unit/simple_assoc_mock_model.mof |
| testconn  | http://localhost | root/cimv2  | usr1   |        30 | True     |                                        |
| testconn2 | http://localhost | root/cimv2  | usr1   |        30 | True     |                                        |
+-----------+------------------+-------------+--------+-----------+----------+----------------------------------------+
pywbemcli>
```

Issues:
1. The "connection save" command in interactive mode as currently implemented seems to use the current connection **of the session** and overwrites that with its connection related general options. It fails because there was no current connection in the interactive session. I think that should be changed so that it saves **its own** current connection (i.e. the result of parameters inherited from the session, and specified on its own command line).
2. Issue in error message "Error: No current connection connection".

**Comment by KSchopmeyer** [comment]
I agree with issue.  Will sort out.

**Comment by andy-maier** [comment]
Fixed in PR #403.
